src/main/java/com/esri/samples/map/read_geopackage/ReadGeoPackageSample.java

```diff
@@ -0,0 +1,190 @@
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.map.read_geopackage;
+
+import java.io.File;
+
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.input.MouseButton;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+import javafx.util.Callback;
+
+import com.esri.arcgisruntime.data.GeoPackage;
+import com.esri.arcgisruntime.data.GeoPackageFeatureTable;
+import com.esri.arcgisruntime.layers.FeatureLayer;
+import com.esri.arcgisruntime.layers.Layer;
+import com.esri.arcgisruntime.layers.RasterLayer;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.raster.GeoPackageRaster;
+
+public class ReadGeoPackageSample extends Application {
+
+  private MapView mapView;
+
+  @Override
+  public void start(Stage stage) throws Exception {
+
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+      scene.getStylesheets().add(getClass().getResource("/css/style.css").toExternalForm());
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Read GeoPackage Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // create a map and add it to the map view
+      final ArcGISMap map = new ArcGISMap(Basemap.Type.STREETS, 39.7294, -104.8319, 11);
+      mapView = new MapView();
+      mapView.setMap(map);
+
+      // create a control panel
+      VBox vBoxControl = new VBox(6);
+      vBoxControl.setMaxSize(250, 260);
+      vBoxControl.getStyleClass().add("panel-region");
+
+      // create two list views, one showing the layers in the map,
+      // the other, showing the layers in the geoPackage not yet added to the map
+      ListView<Layer> mapLayers = new ListView<>();
+      ListView<Layer> geoPackageLayers = new ListView<>();
+
+      // create labels for the lists
+      Label mapLayersLabel = new Label("Map layers");
+      Label geoPackageLayersLabel = new Label("GeoPackage layers (not in the map)");
+      mapLayersLabel.getStyleClass().add("panel-label");
+      geoPackageLayersLabel.getStyleClass().add("panel-label");
+
+      // add labels and lists to the control panel
+      vBoxControl.getChildren().addAll(mapLayersLabel, mapLayers, geoPackageLayersLabel, geoPackageLayers);
```

Comment:
Code move code below to here so that they are closer together and easier to read.
```java
 // create a control panel
      VBox vBoxControl = new VBox(6);
      vBoxControl.setMaxSize(250, 260);
      vBoxControl.getStyleClass().add("panel-region");
```